catalogue/webapp/test/e2e/actions/images.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  colourSelector,
+  colourSelectorFilterDropDown,
+  modalexpandedImaged,
+  modalexpandedImageViewMoreButton,
+} from '../selectors/images';
+
+export async function clickActionColourDropDown(): Promise<void> {
+  await page.click(colourSelectorFilterDropDown);
+}
+
+export async function clickActionColourPicker(): Promise<void> {
+  await page.click(colourSelector, {
+    position: { x: 100, y: 100 },
```

Comment:
Is this to select a specific colour?

Reply:
no this is just picking the centre for any colour.